### ISSUE TITLE
FreeBSD Uptime/Load 

### ISSUE THREAD

**Issue by phillhocking** [gpakosz/.tmux#280]
It took me a bit of doing in order to discover how best to display uptime and load on a FreeBSD machine. 

![image](https://user-images.githubusercontent.com/4030898/64238770-e18b9580-ceb3-11e9-9e5f-1e357a5fcf59.png)

I was having a hard time with the complicated expression evaluating within the brackets on the .tmux.conf.local so I just had it call this script which a helpful individual in #freebsd on freenode helped me out with: 

`#!/usr/local/bin/bash`
`expr "$(uptime --libxo xml)" : '.*<uptime-human>\([^<]*\),</uptime-human>'`

I learned this is best for load: `sh -s |sysctl vm.loadavg |cut -c 14-28`

I hope these suggestions can make it into the uptime section of the main .tmux.conf where it uses uname -s to derive the distribution to select uptime. I am not good enough at coding to tackle that so I just put in an override, but I hope this suggestion can make it into the main repository! 

I am very pleased with this configuration and appreciate all the time and effort that went into it. Cheers!

**Comment by gpakosz** [comment]
Hello @phillhocking 👋

I'll have a look!

**Comment by gpakosz** [comment]
If I understand correctly, you want a totally different display than what the `#{uptime_*}` variables provide.

If that's the case, I suggest that you simply don't use them. Instead, you can put your own script right in the `tmux_conf_theme_status_left` like you did.

**Comment by phillhocking** [comment]
Hey @gpakosz sorry for the delay in reply; I was not able to see what the regular visualizations are for {uptime_*} and #{loadavg} due to them not working in a FreeBSD environment; I can probably spin it up on Linux and see what it looks like there. I imagine that the default visualizations work and look a lot better than what I cobbled together, I just am not particularly skilled at coding/shell scripting so I did what I could to get the information without parsing it down into minutes/seconds/days etc. 

I would be more than willing to test if that would be helpful!
